pom.xml

```diff
@@ -131,6 +131,16 @@
       <id>repo.jenkins-ci.org</id>
       <url>http://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>twitter</id>
+      <url>http://maven.twttr.com</url>
```

Comment:
ah ok - in that case - I think this is ok as long as it builds. I think Tyler is ok with it - so probably kosher? I am being a little conservative here ;)